ckan/logic/validators.py

```diff
@@ -607,8 +607,9 @@ def user_password_not_empty(key, data, errors, context):
             errors[key].append(_('Missing value'))
 
 def user_about_validator(value,context):
-    if 'http://' in value or 'https://' in value:
-        raise Invalid(_('Edit not allowed as it looks like spam. Please avoid links in your description.'))
+    if not config.get('ckan.markdown_filter_links', False):
```

Comment:
Let's keep the old default. It makes sense for sites that allow anonymous users to sign up